### PR TITLE
OSDOCS-10889 OCP 4.14.30 Release Notes

### PR DESCRIPTION
**Pull request by cbippley**
Version(s): 4.14

Issue: [OSDOCS-10889](https://issues.redhat.com/browse/OSDOCS-10889)

[Link to docs preview](https://77589--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-14-release-notes.html#ocp-4-14-30)

QE review: Not needed

Additional information: This release ships June 19th, errata links will not work until then.

### CODE REVIEW

File: release_notes/ocp-4-14-release-notes.adoc [hunk 1]
```diff
@@ -3322,7 +3322,52 @@ This section will continue to be updated over time to provide notes on enhanceme
...
+
+[source,terminal]
+----
+$ oc adm release info 4.14.30 --pullspecs
+----
+
+[id="ocp-4-14-30-bug-fixes"]
+==== Bug fixes
+
+* Previously,  (link:https://issues.redhat.com/browse/OCPBUGS-35085[*OCPBUGS-35085*])
```
Comment (obrown1205) [inline release_notes/ocp-4-14-release-notes.adoc:3344]: * Previously, the OpenShift Agent Installer reported installed SATA SDDs as removable and refused to use any of them as installation targets. With this release removable disks are eligible for installation and the issue has been resolved. (link:https://issues.redhat.com/browse/OCPBUGS-35085[*OCPBUGS-35085*])

File: release_notes/ocp-4-14-release-notes.adoc [hunk 2]
```diff
@@ -3322,7 +3322,52 @@ This section will continue to be updated over time to provide notes on enhanceme
...
+
+* https://issues.redhat.com/browse/OCPBUGS-35034
```
Comment (obrown1205) [inline release_notes/ocp-4-14-release-notes.adoc:3346]: Enhancement!

[id="ocp-4-15-18-password-colon-character"]
===== Allowing pull secret passwords to contain a colon character

* This release adds the ability to include a colon character in your password for the Openshift Assisted Installer.

File: release_notes/ocp-4-14-release-notes.adoc [hunk 3]
```diff
@@ -3322,7 +3322,52 @@ This section will continue to be updated over time to provide notes on enhanceme
...
+
+* Previously, an AWS policy issue prevented the Cluster API Provider for AWS from retrieving the necessary domain information. As a consequence, installing an AWS hosted cluster with a custom domain failed. With this update, the policy issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-34856[*OCPBUGS-34856*])
+
+* https://issues.redhat.com/browse/OCPBUGS-34814
```
Comment (obrown1205) [inline release_notes/ocp-4-14-release-notes.adoc:3350]: * Previously, when a cluster or BareMetal Host (BMH) was deleted, a `PreprovImage` image was created during the _powering off before delete_ stage. As a result, the cluster resource got stuck. With this release, an exception was created for the _powering off before delete_ stage and the issue has been resolved. (link:https://issues.redhat.com/browse/OCPBUGS-34814[OCPBUGS-34814]).

File: release_notes/ocp-4-14-release-notes.adoc [hunk 4]
```diff
@@ -3322,7 +3322,52 @@ This section will continue to be updated over time to provide notes on enhanceme
...
+
+* https://issues.redhat.com/browse/OCPBUGS-34792 RHE
```
Comment (obrown1205) [inline release_notes/ocp-4-14-release-notes.adoc:3352]: Still looks private here

File: release_notes/ocp-4-14-release-notes.adoc [hunk 5]
```diff
@@ -3322,7 +3322,52 @@ This section will continue to be updated over time to provide notes on enhanceme
...
+
+* https://issues.redhat.com/browse/OCPBUGS-34668
```
Comment (obrown1205) [inline release_notes/ocp-4-14-release-notes.adoc:3354]: Enhancement:
Previously, when `virtualHostedStyle` with the `regionEndpoint` was set in the image registry operator config, the image registry ignored the virtual hosted style config and failed to start. This release adds a new upstream distribution configuration (force-path style) in favor of the downstream-only version (virtual-hosted style). (link:https://issues.redhat.com/browse/OCPBUGS-34668[OCPBUGS-34668])

File: release_notes/ocp-4-14-release-notes.adoc [hunk 6]
```diff
@@ -3322,7 +3322,52 @@ This section will continue to be updated over time to provide notes on enhanceme
...
+
+* https://issues.redhat.com/browse/OCPBUGS-34657
```
Comment (obrown1205) [inline release_notes/ocp-4-14-release-notes.adoc:3356]: * Previously, the 'metal3-ironic' and 'metal3-ironic-inspector' pods failed when upgrading to {product-title} 4.15.11 from 4.15.8 due to an install failure related to FIPS mode enablement. With this release, the issue has been resolved. (link:[https://issues.redhat.com/browse/OCPBUGS-34657[OCPBUGS-34657]).

File: release_notes/ocp-4-14-release-notes.adoc [hunk 7]
```diff
@@ -3322,7 +3322,52 @@ This section will continue to be updated over time to provide notes on enhanceme
...
+
+* Previously, certain situations caused transfer of an Egress IP address from one node to a different node to fail, and this failure impacted the OVN-Kubernetes network. The network failed to send gratuitous Address Resolution Protocol (ARP) requests to peers to inform them of the new node’s medium access control (MAC) address. As a result, peers would temporarily send reply traffic to an old node and this traffic led to failover issues. With this release, the OVN-Kubernetes network correctly sends a gratuitous ARP to peers to inform them of the new Egress IP node MAC address, so that each peer can send reply traffic to the new node without causing failover time issues. (link:https://issues.redhat.com/browse/OCPBUGS-34570[*OCPBUGS-34570*])
+
+* Previously, the `wait-for-ceo` command used during the bootstrapping process to verify etcd rollout would not report errors for some failure modes. With this release, those error messages should now be visible on the `bootkube` script if the command exits in an error case. (link:https://issues.redhat.com/browse/OCPBUGS-34495[*OCPBUGS-34495*])
+
+* https://issues.redhat.com/browse/OCPBUGS-5943
```
Comment (obrown1205) [inline release_notes/ocp-4-14-release-notes.adoc:3362]: * Previously, the DNS Operator did not verify that a cluster had ready nodes with available CPU in at least two availability zones, and the DNS daemon set did not use surge for rolling updates. As a result, clusters in which all nodes were in the same availability zone would repeatedly emit `TopologyAwareHintsDisabled` events for the cluster DNS service. With this release, the `TopologyAwareHintsDisabled` events are no longer emitted on clusters that do not have nodes in multiple availability zones and the issue has been resolved. (link:[https://issues.redhat.com/browse/OCPBUGS-5943[OCPBUGS-5943])

File: release_notes/ocp-4-14-release-notes.adoc [hunk 8]
```diff
@@ -3322,7 +3322,52 @@ This section will continue to be updated over time to provide notes on enhanceme
...
+
+* https://issues.redhat.com/browse/OCPBUGS-34657
+
+* Previously, certain situations caused transfer of an Egress IP address from one node to a different node to fail, and this failure impacted the OVN-Kubernetes network. The network failed to send gratuitous Address Resolution Protocol (ARP) requests to peers to inform them of the new node’s medium access control (MAC) address. As a result, peers would temporarily send reply traffic to an old node and this traffic led to failover issues. With this release, the OVN-Kubernetes network correctly sends a gratuitous ARP to peers to inform them of the new Egress IP node MAC address, so that each peer can send reply traffic to the new node without causing failover time issues. (link:https://issues.redhat.com/browse/OCPBUGS-34570[*OCPBUGS-34570*])
+
+* Previously, the `wait-for-ceo` command used during the bootstrapping process to verify etcd rollout would not report errors for some failure modes. With this release, those error messages should now be visible on the `bootkube` script if the command exits in an error case. (link:https://issues.redhat.com/browse/OCPBUGS-34495[*OCPBUGS-34495*])
```
Comment (obrown1205) [inline release_notes/ocp-4-14-release-notes.adoc:3360]: Replace "should" with "are"

File: release_notes/ocp-4-14-release-notes.adoc [hunk 9]
```diff
@@ -3322,7 +3322,61 @@ This section will continue to be updated over time to provide notes on enhanceme
...
+
+* Previously, an AWS policy issue prevented the Cluster API Provider for AWS from retrieving the necessary domain information. As a consequence, installing an AWS hosted cluster with a custom domain failed. With this update, the policy issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-34856[*OCPBUGS-34856*])
+
+* Previously, when a cluster or BareMetal Host (BMH) was deleted, a `PreprovImage` image was created during the powering off before delete stage. As a result, the cluster resource got stuck. With this release, an exception is created for the powering off before delete stage and the issue has been resolved. (link:https://issues.redhat.com/browse/OCPBUGS-34814[*OCPBUGS-34814*]).
+
+* Previously, when `virtualHostedStyle` with the `region` endpoint was set in the image registry operator config, the image registry ignored the virtual hosted style config and failed to start. This release adds a new upstream distribution configuration (force-path style) in favor of the downstream-only version (virtual-hosted style). (link:https://issues.redhat.com/browse/OCPBUGS-34668[*OCPBUGS-34668*])
```
Comment (xenolinux) [inline release_notes/ocp-4-14-release-notes.adoc:3365]:
```suggestion
* Previously, when you enabled `virtualHostedStyle` with `regionEndpoint` in the Image Registry Operator configuration, the image registry ignored `virtualHostedStyle` and failed to start. This release adds a new upstream distribution configuration in favor of the downstream-only version. (link:https://issues.redhat.com/browse/OCPBUGS-34668[*OCPBUGS-34668*])
```

Comment (xenolinux) [reply release_notes/ocp-4-14-release-notes.adoc:3365]: - Is `virtualHostedStyle` a parameter we can specify that in the text.
- I am not sure what this means "This release adds a new upstream distribution configuration in favor of the downstream-only version." 
- Please confirm the values in back ticks. For example, if it's `region` endpoint or `regionEndpoint`

File: release_notes/ocp-4-14-release-notes.adoc [hunk 10]
```diff
@@ -3322,7 +3322,61 @@ This section will continue to be updated over time to provide notes on enhanceme
...
+
+[id="ocp-4-14-30-bug-fixes"]
+==== Bug fixes
+
+* Previously, the OpenShift Agent Installer reported installed SATA SDDs as removable and refused to use any of them as installation targets. With this release removable disks are eligible for installation and the issue has been resolved. (link:https://issues.redhat.com/browse/OCPBUGS-35085[*OCPBUGS-35085*])
+
+* Previously, an AWS policy issue prevented the Cluster API Provider for AWS from retrieving the necessary domain information. As a consequence, installing an AWS hosted cluster with a custom domain failed. With this update, the policy issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-34856[*OCPBUGS-34856*])
+
+* Previously, when a cluster or BareMetal Host (BMH) was deleted, a `PreprovImage` image was created during the powering off before delete stage. As a result, the cluster resource got stuck. With this release, an exception is created for the powering off before delete stage and the issue has been resolved. (link:https://issues.redhat.com/browse/OCPBUGS-34814[*OCPBUGS-34814*]).
```
Comment (xenolinux) [inline release_notes/ocp-4-14-release-notes.adoc:3363]:
```suggestion
* Previously, when you deleted a cluster or BareMetal Host (BMH), a `PreprovImage` image was created during cluster deletion. As a result, the cluster resources were stuck. With this release, an exception is created for the powering off before delete stage and the issue has been resolved. (link:https://issues.redhat.com/browse/OCPBUGS-34814[*OCPBUGS-34814*]).
```

- Is there any particular state, cluster resource were stuck in?
- Can we explain what exception is given for powering off?

File: release_notes/ocp-4-14-release-notes.adoc [hunk 11]
```diff
@@ -3322,7 +3322,61 @@ This section will continue to be updated over time to provide notes on enhanceme
...
+
+* Previously, the `metal3-ironic` and `metal3-ironic-inspector` pods failed when upgrading to {product-title} 4.15.11 from 4.15.8 due to an install failure related to FIPS mode enablement. With this release, the issue has been resolved. (link:https://issues.redhat.com/browse/OCPBUGS-34657[*OCPBUGS-34657*]).
+
+* Previously, certain situations caused transfer of an Egress IP address from one node to a different node to fail, and this failure impacted the OVN-Kubernetes network. The network failed to send gratuitous Address Resolution Protocol (ARP) requests to peers to inform them of the new node’s medium access control (MAC) address. As a result, peers would temporarily send reply traffic to an old node and this traffic led to failover issues. With this release, the OVN-Kubernetes network correctly sends a gratuitous ARP to peers to inform them of the new Egress IP node MAC address, so that each peer can send reply traffic to the new node without causing failover time issues. (link:https://issues.redhat.com/browse/OCPBUGS-34570[*OCPBUGS-34570*])
```
[the remark this answers is not in the document]
Comment (xenolinux) [reply release_notes/ocp-4-14-release-notes.adoc:3369]: I have given my suggestion, but this might need revising.

**Review by xenolinux — COMMENTED** [review]
A couple of suggestions, revisions to consider. Thanks!

/remove-label peer-review-needed
/remove-label peer-review-in-progress
/label peer-review-done

File: release_notes/ocp-4-14-release-notes.adoc [hunk 12]
```diff
@@ -3322,7 +3322,61 @@ This section will continue to be updated over time to provide notes on enhanceme
...
+
+[source,terminal]
+----
+$ oc adm release info 4.14.30 --pullspecs
+----
+
+[id="ocp-4-14-30-enhancements"]
+==== Enhancements
+
+The following enhancement is included in this z-stream release:
```
Comment (jeana-redhat) [inline release_notes/ocp-4-14-release-notes.adoc:3344]:
```suggestion
The following enhancements are included in this z-stream release:
```

File: release_notes/ocp-4-14-release-notes.adoc [hunk 13]
```diff
@@ -3322,7 +3322,61 @@ This section will continue to be updated over time to provide notes on enhanceme
...
+===== Configuring the topology feature for the Cinder CSI Driver
+
+Previously, the topology feature for the Cinder CSI Driver was always active, because you could not disable the topology feature. With this release, the topology feature is based on the compute and storage availability zones and you can disable the topology feature. (link:https://issues.redhat.com/browse/OCPBUGS-34792[*OCPBUGS-34792*])
+
+[id="ocp-4-14-30-bug-fixes"]
+==== Bug fixes
+
+* Previously, the {product-title} {ai-full} reported installed SATA SDDs as removable and did not use any of them as installation targets. With this release, removable disks are eligible for installation and the issue has been resolved. (link:https://issues.redhat.com/browse/OCPBUGS-35085[*OCPBUGS-35085*])
+
+* Previously, an Amazon Web Services (AWS) policy issue prevented the Cluster API Provider for AWS from retrieving the necessary domain information. As a consequence, installing an AWS hosted cluster with a custom domain failed. With this update, the policy issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-34856[*OCPBUGS-34856*])
```
Comment (jeana-redhat) [inline release_notes/ocp-4-14-release-notes.adoc:3361]: Normally would not hold this for the use of attributes, but the `Cluster API Provider AWS` one replaces incorrect text (there is no "for" in the phrase)
```suggestion
* Previously, an {aws-first} policy issue prevented the {cap-aws-short} from retrieving the necessary domain information. As a consequence, installing an {aws-short} hosted cluster with a custom domain failed. With this update, the policy issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-34856[*OCPBUGS-34856*])
```

File: release_notes/ocp-4-14-release-notes.adoc [hunk 14]
```diff
@@ -3322,7 +3322,61 @@ This section will continue to be updated over time to provide notes on enhanceme
...
+
+* Previously, when you deleted a cluster or BareMetal Host (BMH), a `PreprovImage` image was created during cluster deletion. As a result, the cluster resources were stuck. With this release, an exception is created for the powering off before delete stage and the issue has been resolved. (link:https://issues.redhat.com/browse/OCPBUGS-34814[*OCPBUGS-34814*]).
```
Comment (jeana-redhat) [inline release_notes/ocp-4-14-release-notes.adoc:3363]:
```suggestion
* Previously, when you deleted a cluster or BareMetal Host (BMH), a `PreprovImage` image was created during cluster deletion. As a result, the cluster resources were stuck. With this release, an exception is created for the powering off before the delete stage and the issue has been resolved. (link:https://issues.redhat.com/browse/OCPBUGS-34814[*OCPBUGS-34814*]).
```

File: release_notes/ocp-4-14-release-notes.adoc [hunk 15]
```diff
@@ -3322,7 +3322,61 @@ This section will continue to be updated over time to provide notes on enhanceme
...
+
+* Previously, the OVN-Kubernetes network plugin could not send gratuitous Address Resolution Protocol (ARP) requests to peers to inform them about the medium access control (MAC) address of a new node, because some situations caused the transfer of the Egress IP address from one node to another to fail.  This resulted in failover issues. With this release, the OVN-Kubernetes network plugin correctly informs peers about the medium access control (MAC) address of a new node, without causing a failover issue. (link:https://issues.redhat.com/browse/OCPBUGS-34570[*OCPBUGS-34570*])
+
+* Previously, if you used the `wait-for-ceo` command during the bootstrapping process, the command would not report the error messages in case of a failure. With this release,  the command reports the error messages in the `bootkube` script for you to view. (link:https://issues.redhat.com/browse/OCPBUGS-34495[*OCPBUGS-34495*])
+
+* Previously, the installation program did not support the `ca-west-1` Amazon Web Services (AWS) region. With this release, the `ca-west-1` region is supported and the issue has been resolved. (link:https://issues.redhat.com/browse/OCPBUGS-34024[*OCPBUGS-34024*])
```
Comment (jeana-redhat) [inline release_notes/ocp-4-14-release-notes.adoc:3373]:
```suggestion
* Previously, the installation program did not support the `ca-west-1` {aws-first} region. With this release, the `ca-west-1` region is supported and the issue has been resolved. (link:https://issues.redhat.com/browse/OCPBUGS-34024[*OCPBUGS-34024*])
```

**Review by jeana-redhat — COMMENTED** [review]
A few small nits here. The only one that should definitely be addressed for accuracy is replacing "Cluster API Provider for AWS" with `{cap-aws-short}`. Feel free to DM me directly for a merge :)

/remove-label merge-review-in-progress
/remove-label merge-review-needed

**Comment by bscott-rh** [comment]
merge review LGTM

**Pull request merged**